src/uid2ApiClient.ts

```diff
@@ -10,6 +12,12 @@ export type SuccessRefreshResult = {
 };
 export type RefreshResult = SuccessRefreshResult | RefreshResultWithoutIdentity;
 
+export type SuccessCstgResult = {
+  status: "success";
+  identity: Uid2Identity;
+};
+export type CstgResult = SuccessCstgResult;
```

Comment:
would it make sense to create a FailedCstgResult? then avoid the `any` on line 137?

Reply:
Yes, we will have to revisit this as the API continues to take shape. I think for a first cut of we can leave it with just a success result.